Flask-Login 0.6.2 Will Break Tests

Latest version of Flask-login, release on June 25th (0.6.2) will break our test suite. 0.6.1 works.

Test
```
pytest tests/server/blueprints/alignviewers/test_alignviewers_views.py::test_igv_not_authorized
```

Will not return unauthorised `403`, as expected, but OK `200`. This will cause the test to fail
```
FAILED tests/server/blueprints/alignviewers/test_alignviewers_views.py::test_igv_not_authorized - assert 200 == 403
```


Suggested different approaches to solve this. 
* Freeze flask-login, only
* Freeze all flask associated packages
* Find bug or workaround

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,9 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Added
 - Documentation on how to create/update PanelApp panels
 - Add filter by local observations (archive) to structural variants filters
+### Fixed
+- Added a not-authorized `auto-login` fixture according to changes in Flask-Login 0.6.2
+- Renamed `cache_timeout` param name of flask.send_file function to `max_age` (Flask 2.2 compliant)
 ### Changed
 - State that loqusdb observation is in current case if observations count is one and no cases are shown  
 - Better pagination and number of variants returned by queries in `Search SNVs and INDELs` page

--- a/scout/server/blueprints/cases/views.py
+++ b/scout/server/blueprints/cases/views.py
@@ -280,7 +280,7 @@ def mt_report(institute_id, case_name):
             mimetype="application/zip",
             as_attachment=True,
             download_name="_".join(["scout", case_name, "MT_report", today]) + ".zip",
-            cache_timeout=0,
+            max_age=0,
         )
 
     shutil.rmtree(temp_excel_dir)

--- a/tests/server/blueprints/alignviewers/test_alignviewers_views.py
+++ b/tests/server/blueprints/alignviewers/test_alignviewers_views.py
@@ -106,7 +106,7 @@ def test_igv_not_authorized(app, user_obj, case_obj, variant_obj):
     # GIVEN an initialized app
     with app.test_client() as client:
 
-        # GIVEN that the user is logged in but not wuthorized to see the page
+        # GIVEN that the user is logged in but not authorized to see the page
         client.get(url_for("auto_login_not_authorized"))
 
         # WHEN the igv endpoint is invoked with the right parameters

--- a/tests/server/blueprints/alignviewers/test_alignviewers_views.py
+++ b/tests/server/blueprints/alignviewers/test_alignviewers_views.py
@@ -103,17 +103,11 @@ def test_remote_cors(app):
 def test_igv_not_authorized(app, user_obj, case_obj, variant_obj):
     """Test view requests and produces igv alignments, when the user dosn't have access to the case"""
 
-    # GIVEN a user that is not an admin nor has access to demo case:
-    store.user_collection.find_one_and_update(
-        {"_id": user_obj["_id"]},
-        {"$set": {"roles": [], "institutes": []}},
-    )
-
     # GIVEN an initialized app
     with app.test_client() as client:
 
-        # GIVEN that the user is logged in
-        client.get(url_for("auto_login"))
+        # GIVEN that the user is logged in but not wuthorized to see the page
+        client.get(url_for("auto_login_not_authorized"))
 
         # WHEN the igv endpoint is invoked with the right parameters
         resp = client.get(

--- a/tests/server/blueprints/alignviewers/test_alignviewers_views.py
+++ b/tests/server/blueprints/alignviewers/test_alignviewers_views.py
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
-import requests
 from flask import session, url_for
 
 from scout.server.extensions import store

--- a/tests/server/conftest.py
+++ b/tests/server/conftest.py
@@ -81,6 +81,14 @@ def app(real_database_name, real_variant_database, user_obj, loqusdburl):
         assert login_user(user_inst, remember=True)
         return "ok"
 
+    @app.route("/auto_login_not_authorized")
+    def auto_login_not_authorized():
+        user_obj["roles"] = []
+        user_obj["institutes"] = []
+        user_inst = LoginUser(user_obj)
+        assert login_user(user_inst, remember=True)
+        return "ok"
+
     return app
 
 

